src/components/SearchField/SearchField.test.tsx

```diff
@@ -247,3 +247,35 @@ test('it calls the provided onClear and onChange after clicking the clear button
   expect(onChange).toHaveNthReturnedWith(1, 'this should be cleared...');
   expect(onChange).toHaveNthReturnedWith(2, '');
 });
+
+test('it can be controlled', async () => {
+  const value = 'chroma';
+  const mockFn = jest.fn();
+  const onChange = jest.fn((event) => event.target.value);
+
+  const props = getBaseProps();
+  const { findByTestId } = renderWithTheme(
+    <SearchField
+      {...props}
+      onSearch={mockFn}
+      data-testid={testId}
+      value={value}
+      onChange={onChange}
+    />
+  );
+
+  const input: any = await findByTestId(testId);
```

Comment:
Yeah, would be better, I was just following the other tests in the file doing the same. It would have to be like:

```
  const input: HTMLInputElement = (await findByTestId(testId)) as any;
```